src/scripts/ui/GrandBazaarBuildingBody.tsx

```diff
@@ -58,7 +63,10 @@ export function GrandBazaarBuildingBody({ gameState, xy }: IBuildingComponentPro
    const allMarketTrades: IGrandBazaarMarketData[] = [];
    marketBuildings?.forEach((tile, xy) => {
       const market = tile.building as IMarketBuildingData;
-      const sellAmount = market.capacity * getMarketSellAmount(xy, gameState);
+      if (market.status === "building" || market.status === "upgrading") {
```

Comment:
Should check `status !== "completed"`. It's more likely to be correct If more statuses are added

Reply:
Done